datanode/gateway/graphql/resolvers.go

```diff
@@ -591,6 +591,88 @@ func (r *myQueryResolver) Erc20WithdrawalApproval(ctx context.Context, wid strin
 	}, nil
 }
 
+func (r *myQueryResolver) Erc20ListAssetBundle(ctx context.Context, assetID string) (*Erc20ListAssetBundle, error) {
+	res, err := r.tradingDataClientV2.GetERC20ListAssetBundle(
+		ctx, &v2.GetERC20ListAssetBundleRequest{AssetId: assetID})
+	if err != nil {
+		return nil, err
+	}
+
+	return &Erc20ListAssetBundle{
+		AssetSource: res.AssetSource,
+		VegaAssetID: res.VegaAssetId,
+		Nonce:       res.Nonce,
+		Signatures:  res.Signatures,
+	}, nil
+}
+
+func (r *myQueryResolver) Erc20MultiSigSignerAddedBundles(ctx context.Context, nodeID string, epochSeq string, pagination *v2.Pagination) (*ERC20MultiSigSignerAddedConnection, error) {
+	res, err := r.tradingDataClientV2.GetERC20MultiSigSignerAddedBundles(
+		ctx, &v2.GetERC20MultiSigSignerAddedBundlesRequest{
+			NodeId:     nodeID,
```

Comment:
I think as of a couple of days ago this now has a `submitter` field too, so we should wire that in too.